ctgan/data_transformer.py

```diff
@@ -47,7 +48,9 @@ def _fit_continuous(self, data):
         """
         column_name = data.columns[0]
         gm = ClusterBasedNormalizer(
-            missing_value_generation='from_column', max_clusters=min(len(data), 10))
+            missing_value_generation='from_column',
+            max_clusters=min(len(data), self._max_clusters)
```

Comment:
Unrelated bug, we should pass `ClusterBasedNormalizer` `self._max_clusters` and not 10.